app/models/location.rb

```diff
@@ -1,13 +1,28 @@
 class Location < ApplicationRecord
+  include Validations::PropertyValidations
```

Comment:
Might also be worth extracting the postcode regexp to somewhere (same place as the postcode normalisation maybe?) and including that instead since it's a bit misleading to include the PropertyValidations module here when we're not using any of the actual validations.